cfme/containers/provider/openshift.py

```diff
@@ -412,3 +441,31 @@ def is_refreshed(self, refresh_timer=None, refresh_delta=600):
             )
         finally:
             return valid
+
+    def enable_provider_alerts_collection(self):
```

Comment:
These two methods (enable_provider_alerts_collection and enable_node_alerts_collection)are exactly the same, I think they should be a single method that takes an argument, or even better a list of arguments, for what type of alert profiles to assign.

Reply:
@mshriver I added a third function which getting a profile and assigns it to 'The Enterprise'.

This function supports both single and multiple profiles, I didn't use the multiple version because I enable each profile in various function, and I want to keep this structure.

Reply:
https://github.com/shalomnaim1/integration_tests/blob/a933c585d2530f31569b7f14b44f7d8040bbc6a2/cfme/containers/provider/openshift.py#L448-L458